sklearn/tests/test_random_projection.py

```diff
@@ -206,13 +206,6 @@ def test_sparse_random_matrix():
 ###############################################################################
 
 
-@pytest.mark.parametrize("density", [1.1, 0, -0.1])
-def test_sparse_random_projection_transformer_invalid_density(density):
-    for RandomProjection in all_SparseRandomProjection:
-        with pytest.raises(ValueError):
-            RandomProjection(density=density).fit(data)
-
-
 @pytest.mark.parametrize("n_components, fit_data", [("auto", [[0, 1, 2]]), (-10, data)])
```

Comment:
I believe we don't need to test the second case of the parametrization anymore since that'll be delegated to the `param_validation` system.